route-rs-runtime/src/utils/test/packet_collectors.rs

```diff
@@ -3,6 +3,8 @@ use crossbeam::crossbeam_channel::Sender;
 use futures::{Async, Future, Poll};
 use std::fmt::Debug;
 
+/// A structure that may be handed an input stream that it will exhaustively drain from until it
+/// recieves a None. Useful for testing purposes.
```

Comment:
Can remove "Useful for testing purposes". It exists in `utils/test` so I users can pick up on the context.